### DISCUSSION TITLE
WDD/

### DISCUSSION THREAD

**Discussion by giscus** [SungbinYang/SungbinYang.github.io#15] [Comments]
# WDD/

양성빈의 블로그

https://sungbin.kr/WDD/

**Comment by lsb530** [comment]
크 WDD 너무 좋은 말인 것 같습니다! 이제 앞으로 "WDD by Sungbin"이 되겠군요!ㅎㅎ

**Reply by SungbinYang** [reply]
답변 감사합니다!

**Comment by BEOKS** [comment]
WDD 정말 좋은 개념인것 같습니다:) 본질에 대한 탐구는 튼튼한 기초와 올바른 방향으로 나아가기 위한 밑거름이니까요. 그렇다면 어디까지 왜?라고 질문해야할까요? 왜?라는 질문은 끝이없는 재귀함수와 같은데 멈춰야하는 기준이 있을까요?

**Reply by SungbinYang** [reply]
그쵸! 재귀함수처럼 무조건 왜라는 질문만 던져서는 안됩니다! 멈춰야 하는 기준이라면 상황마다 다르겠지만 저라면 너무 깊은 곳까지 왜를 질문한다싶으면 멈추곤 합니다!
